src/psyclone/tests/psyir/frontend/fparser2_parameter_stmts_test.py

```diff
@@ -221,6 +221,9 @@ def test_unsupported_parameter_statements_produce_codeblocks(fortran_reader,
     # statements
     code = fortran_writer(psyir)
     assert code == '''\
+! PSyclone CodeBlock reason:
+!  - Could not parse 'PARAMETER(var1 = 'hello')' because 'var1' has an \
```

Comment:
Not strictly this PR but since it now makes this message public, I don't think 'parse' is the right word here as that implies it's an fparser problem. Perhaps 'process' instead?

Reply:
Done